src/storage.ts

```diff
@@ -190,3 +191,25 @@ async function createAwardsCollection(appDb: Db) {
 
     return { Awards, saveAward, getAwards };
 }
+
+async function createAdminCollection(appDb:Db) {
+    const Admins = appDb.collection<TeamBotUser>("Admins");
```

Comment:
TeamBotUser is actually the bot itself in the team. Slack creates a bot user (e.g. @​​memelord) in each team that the app is installed into.